requirements.yml

```diff
@@ -1 +1,2 @@
 ---
+- src: kwoodson.yedit
```

Comment:
Please add ansible.utils here as well

Reply:
Some information about kwoodson.yedit collection being required would be nice in the readme (Unless it is there and I simply missed it)

Reply:
Hi @JakubKotkowiak!  I removed the `kwoodson.yedit` module altogether for concerns about licensing and I created an embedded `merge_yaml.py` module instead.  Please have a look.